src/wast-parser.cc

```diff
@@ -2146,7 +2146,11 @@ Result WastParser::ParsePlainInstr(std::unique_ptr<Expr>* out_expr) {
           return Result::Error;
         }
 
+#if WABT_BIG_ENDIAN
```

Comment:
(side note: arguably this should just write the v128 directly, instead of doing this and then bitcasting. but it was done like this and there's probably a reason it was done like this but we personally don't know that reason and we don't think this makes sense. .-.)

Reply:
If you switch this over to writing to the v128 directly, do any tests fail? I agree that this looks a little weird as it is.